Jummbox/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "author": {
+    "name": "HiddenRealm",
+    "id": "444628005646499851"
+  },
+  "url": "jummbus.bitbucket.io/",
+  "description": {
+    "english": "English"
+  },

```

Comment:
Description should be:
"description": {
    "en": "DESCRIPTION OF WHAT THE SERVICE IS"
}